doc/classes/ProjectSettings.xml

```diff
@@ -1035,23 +1035,28 @@
+			Number of cubemaps to store in reflection atlas. The number of [ReflectionProbe]s in a scene will be limited by this amount. A higher number requires more VRAM.
 		</member>
-		<member name="rendering/quality/reflection_atlas/reflection_size" type="int" setter="" getter="" default="256">
+		<member name="rendering/quality/reflection_atlas/reflection_size" type="int" setter="" getter="" default="128">
+			Size of cubemap faces for [ReflectionProbe]s. A higher number requires more VRAM and may make reflection probe updating slower.
 		</member>
 		<member name="rendering/quality/reflection_atlas/reflection_size.mobile" type="int" setter="" getter="" default="128">
+			Size of cubemap faces for [ReflectionProbe]s when using mobile.
+		</member>
+		<member name="rendering/quality/reflections/fast_filter_high_quality" type="bool" setter="" getter="" default="false">
+			Use a higher quality variant of the fast filtering algorithm. Significantly slower than using default quality, but results in smoother reflections. Should only be used when scene is especially detailed.
 		</member>
 		<member name="rendering/quality/reflections/ggx_samples" type="int" setter="" getter="" default="1024">
+			Sets the number of samples to take when using importance sampling for [Sky]s and [ReflectionProbe]s. A higher value will result in smoother, higher quality reflections, but increases time to calculate radiance maps. In general, fewer samples are needed for simpler, low dynamic range environments while more samples are needed for HDR environments and environments with a high level of detail.
 		</member>
 		<member name="rendering/quality/reflections/ggx_samples.mobile" type="int" setter="" getter="" default="128">
-		</member>
-		<member name="rendering/quality/reflections/ggx_samples_realtime" type="int" setter="" getter="" default="64">
-		</member>
-		<member name="rendering/quality/reflections/ggx_samples_realtime.mobile" type="int" setter="" getter="" default="16">
+			Sets the number of samples to take when using importance sampling on mobile devices.
 		</member>
 		<member name="rendering/quality/reflections/roughness_layers" type="int" setter="" getter="" default="6">
+			Limits the number of layers to use in radiance maps when using importance sampling. A lower number will be slightly faster and take up less VRAM.
 		</member>
 		<member name="rendering/quality/reflections/texture_array_reflections" type="bool" setter="" getter="" default="true">
-			If [code]true[/code], uses texture arrays instead of mipmaps for reflection probes and panorama backgrounds (sky). This reduces jitter noise on reflections, but costs more performance and memory.
+			If [code]true[/code], uses texture arrays instead of mipmaps for reflection probes and panorama backgrounds (sky). This reduces jitter noise and upscaling artifacts on reflections, but is significantly slower to compute and uses [code]roughness_layers[/code] times more memory.
```

Comment:
Could use `[member rendering/quality/reflections/roughness_layers]` instead of `[code]`. Not the prettiest, but it's accurate.